install-and-configure/advanced-configuration/gpu.md

```diff
@@ -0,0 +1,234 @@
+{% endhint %}
+
+To install NFD as a standalone component, follow the deployment guide [here](https://kubernetes-sigs.github.io/node-feature-discovery/stable/deployment/). A quick start command is also shown below. In some cases, you may have taints applied to GPU nodes which must be tolerated by the NFD DaemonSet. It is recommended to use the Helm installation guide to define tolerations if so.
+
+```sh
+# This command uses Kustomize to deploy Kubernetes resources from a specific version.
+# Refer to the NFD releases to choose the latest, or most applicable, version.
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.16.3
+```
+
+Once NFD is installed, ensure one pod is running on your node(s) with NVIDIA GPUs.
+
+```sh
+kubectl -n node-feature-discovery get pods
+```
+
+After a few moments, check the labels of one such node to ensure the `feature.node.kubernetes.io/pci-10de.present="true"` label has been applied.
+
+```sh
+kubectl get no <my_node_name> -o yaml | yq .metadata.labels
+```
+
+An abridged output of the labels written to an EKS node is shown below.
+
+```yaml
+<snip>
+feature.node.kubernetes.io/kernel-version.full: 5.10.219-208.866.amzn2.x86_64
+feature.node.kubernetes.io/kernel-version.major: "5"
+feature.node.kubernetes.io/kernel-version.minor: "10"
+feature.node.kubernetes.io/kernel-version.revision: "219"
+feature.node.kubernetes.io/pci-10de.present: "true"
+feature.node.kubernetes.io/pci-1d0f.present: "true"
+feature.node.kubernetes.io/storage-nonrotationaldisk: "true"
+<snip>
+```
+
+With NFD having successfully discovered NVIDIA PCI devices and assigned the `feature.node.kubernetes.io/pci-10de.present="true"` label, install DCGM Exporter using this label to attract pods to GPU nodes. When following this process on GKE, additional values may be required to successfully run DCGM Exporter. See the [GKE section](#gke) for more details.
+
+<details>
+<summary>values-dcgm.yaml</summary>
+
+```yaml
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: feature.node.kubernetes.io/pci-10de.present
+          operator: In
+          values:
+          - "true"
+serviceMonitor:
+  enabled: false
+```
+
+</details>
+
+Install DCGM Exporter using the values defined.
+
+```sh
+helm upgrade -i dcgm dcgm-exporter \
+  --repo https://nvidia.github.io/dcgm-exporter/helm-charts \
+  -n dcgm-exporter --create-namespace \
+  -f values-dcgm.yaml
+```
+
+Ensure the DCGM Exporter Pods are in a running state and only on the nodes with NVIDIA GPUs.
+
+```sh
+kubectl -n dcgm-exporter get pods
+```
+
+## Validation
+
+To validate your DCGM Exporter configuration, port-forward into the DCGM Exporter service and ensure first that metrics are being exposed.
```

Comment:
Can we add a ticket to add a diagnostic here please?

Reply:
Yes, this has been a part of the PRD but I've also logged a task separately for 2.4.0 as ENG-2540.